### PR TITLE
US 1583733 Add missing language IDs - 60

### PR DESCRIPTION
**Pull request by v-thpra**
US 1583733 - Fix CATS P1 issues by adding missing language IDs to code blocks.
- Used "sql" for all Entity SQL syntax and code examples.

Contributes to #2192

### CODE REVIEW

**Review by Youssef1313 — CHANGES_REQUESTED** [review]
In the `oftype-entity-sql.md`, the last snippet just before the `See also` section is sql not csharp.

Same with:

- `not-equal-to-entity-sql.md`
- `not-entity-sql.md`
- `negative-entity-sql.md`
- `navigate-entity-sql.md`
- `named-type-constructor-entity-sql.md`
- `multiset-entity-sql.md`
- `multiply-entity-sql.md`
- `modulo-entity-sql.md`
- `limit-entity-sql.md`
- `like-entity-sql.md`
- `less-than-or-equal-to-entity-sql.md`
- `less-than-entity-sql.md`
- `key-entity-sql.md`
- `isof-entity-sql.md`
- `isnull-entity-sql.md`
- `intersect-entity-sql.md`

**Comment by v-thpra** [comment]
@Youssef1313 Thanks. I made your suggested changes.

File: docs/framework/data/adonet/ef/language-reference/intersect-entity-sql.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ expression INTERSECT expression
   
 2. Pass the following query as an argument to the `ExecuteStructuralTypeQuery` method:  
   
- [!code-csharp[DP EntityServices Concepts 2#INTERSECT](../../../../../../samples/snippets/csharp/VS_Snippets_Data/dp entityservices concepts 2/cs/entitysql.cs#intersect)]  
+ [!code-sql[DP EntityServices Concepts 2#INTERSECT](../../../../../../samples/snippets/csharp/VS_Snippets_Data/dp entityservices concepts 2/cs/entitysql.cs#intersect)]
```
Comment (mairaw) [inline docs/framework/data/adonet/ef/language-reference/intersect-entity-sql.md]: You'll also need to change the path for all these examples.

```suggestion
 [!code-sql[DP EntityServices Concepts 2#INTERSECT](~/samples/snippets/tsql/VS_Snippets_Data/dp entityservices concepts 2/tsql/entitysql.cs#intersect)]  
```

Build doesn't say anything but they are blank as is:
image

**Review by mairaw — CHANGES_REQUESTED** [review]
Great set of changes. Just change the path on all the samples that we changed from C# to sql.

File: docs/framework/data/adonet/ef/language-reference/intersect-entity-sql.md (hunk 1 above)
Comment (v-thpra) [reply docs/framework/data/adonet/ef/language-reference/intersect-entity-sql.md]: Fixed the example paths so that they now point to the SQL examples.

**Comment by v-thpra** [comment]
@mairaw Thanks. I made the suggested changes.

**Review by mairaw — APPROVED** [review]
Great changes @v-thpra! Thanks for also improving the existing ones.

**Pull request merged**